### ISSUE TITLE
Swipe compatibility

### ISSUE THREAD

**Issue by didoweb** [iosscripts/iosslider#278]
Hi, I'm making a web app and I'm using Iosslider for caoursel.
It' work very good but I've see the Swipe event is not fired in some Android version and Window Mobile.
In my LG Optimus L9 Android 4.0.4 works great, instead in Galaxy Nexus 4.2.2  swipe doesn't work.

There's a solution?!

Thnak's, Davide

**Comment by quoisinc** [comment]
have similar issue

**Comment by marcwhitbread** [comment]
@didoweb I will take a look at the Nexus 4.2.2 issue. But windows mobile has never been tested or supported unfortunately.

**Comment by didoweb** [comment]
Thank you very much, if you have news I'm here, if I can do anything ask to me.

Davide

**Comment by marcwhitbread** [comment]
@didoweb I am testing various emulators and the slider appears to be working on all of them. The android SDK running 4.2.2 on the Galaxy Nexus appears to be working. I don't have the physical Galaxy Nexus hardware so I'm not sure how to fix it.

**Comment by didoweb** [comment]
Hi, this is the mobile web app where i use Iosslider plus jquery mobile:

http://m.contadicastaldi.it/contadi_mobile/

If you want try I'm happy because I don't know why it' doesn't work

thanks for the kindness, Davide

2013/5/29 Marc Whitbread notifications@github.com

**Comment by marcwhitbread** [comment]
@didoweb are the examples in the download package working? or just your implementation?

**Comment by didoweb** [comment]
When I tried it doesn't work. But I give you confirm tomorrow when I' ve
got the device at my work! Have you tried my emobile example!? Tnx a lot
Davide
Il giorno 29/mag/2013 20:16, "Marc Whitbread" notifications@github.com ha
scritto:

**Comment by marcwhitbread** [comment]
@didoweb yes I tried your example but there are extra moving pieces there which would make testing more difficult. Right now I am focusing on the demos in the download package. That way I can rule out any issues with your implementation that might be causing it.

**Comment by didoweb** [comment]
Ok,  you are right! Then tomorrow I write you about the example, Great,
Davide
Il giorno 29/mag/2013 20:24, "Marc Whitbread" notifications@github.com ha
scritto:

**Comment by didoweb** [comment]
Hi, good morning.

I've trying the demos whit the Nexus 4.2.2.

With default browser it doesn't work.
Instead with Chrome browser it works good.

Then the problem is in the browser.
Maybe because it not support css tranform.
Or something in the gesture.

If you have news I'm always here.

Thnak's a lot, Davide

2013/5/29 Davide Marchet didomarket@gmail.com

**Comment by marcwhitbread** [comment]
@didoweb where are you seeing that it doesn't support css transforms? I am having trouble finding browser specs for the different browsers and different android versions.

**Comment by didoweb** [comment]
"where are you seeing that it doesn't support css transforms?" I've said
something without think. Sorry

2013/5/30 Marc Whitbread notifications@github.com

**Comment by marcwhitbread** [comment]
Odd. I just tested a few devices ranging from 4.0, 4.1.2, 4.2.2 on Samsung/Azus/LG and its working on all of them. Anything unique about your device? Does it have a mouse and keyboard attached perhaps? When you run the benchmarks file in the package, what value do you see for isTouch on the devices in question?

**Comment by didoweb** [comment]
I've not attached anything on this devices, but i suppose it has something
unique in its configuration because I've tested in other devices (old then
this) and it works good. When I've got little more time, I tried to do a
debug.

Thnaks a lot, Davide

2013/5/31 Marc Whitbread notifications@github.com
